packages/manager/modules/pci-universe-components/src/project/index.js

```diff
@@ -1,10 +1,17 @@
 import angular from 'angular';
 import 'ovh-api-services';
 
+import autoGenerateName from './auto-generate-name';
 import regionsList from './regions-list';
+import textAccordion from './text-accordion';
 
 const moduleName = 'ovhManagerPciUniverseComponentsProject';
 
-angular.module(moduleName, ['ovh-api-services', regionsList]);
+angular.module(moduleName, [
+  'ovh-api-services',
```

Comment:
```suggestion
```

Reply:
I think I have to keep this in order to retrieve region list and region quota for ovh-regions-list component